x-pack/plugins/observability/public/pages/alert_details/constants.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const DEFAULT_DATE_FORMAT = 'MMM D, YYYY @ HH:mm:ss.SSS';
```

Comment:
I saw it is defined at many places. When I tried to import from one of the plugins, it was giving error as it was not exposed to use outside of that plugin. I would then have to modify index file of this plugin and export the constant. That's why I created this file. Could you please point me to config which I can use?

Reply:
I don't remember where I saw it, maybe as a helper `formatter`. But If you don't find it or it will take more time, keep it like this and can refactor it later one

Reply:
sure, I will keep an eye on it if I find helper formatter in future, and update it.